### DISCUSSION TITLE
Disk Space Monitor Limit Mount Points...

### DISCUSSION THREAD

**Discussion by stuu3k** [Lissy93/dashy#476] [Q&A] [answered]
Hi Lissy,

Not sure if this exists already, but if not it would be really good to be able to limit to Glances Disk Space Monitor to predefinte Mount Points or Disks in Windows world I guess, please see following example from my NAS, notice how all 4 Mount Points are actually the same volume so it just reports the space 4 times over...
![image](https://user-images.githubusercontent.com/98365629/152648405-c836a39c-f77a-474c-ad8a-2717fdb0fc72.png)

Thanks

Stuart.

**Comment by patilvinit2012** [comment]
Facing the same issue on raspberry pi on portainer: nicolargo/glances:latest-full.
I'm hoping it will show the mounts returned by lsblk.
The Disk IO section shows correct values as expected ie the sda (external drive attatched)


![image](https://github.com/Lissy93/dashy/assets/14797444/13a3c6df-e14c-4baf-80ed-05238ff4834a)

![image](https://github.com/Lissy93/dashy/assets/14797444/c075ae63-b968-464f-8a3a-7534404fcf9f)

![image](https://github.com/Lissy93/dashy/assets/14797444/877ac17e-71f0-4b3a-85df-215d22e5f39a)

@Lissy93 
Is there any option where i can configure which mounts are returned by /api/3/fs?

**Comment by mr-f0xx** [comment]
Same issue and it's annoying as hell lmao

**Comment by CrazyWolf13** [comment, marked as answer]
For anyone finding this later on, here is a solution:

https://github.com/Lissy93/dashy/issues/1333
